src/olympia/versions/migrations/0009_auto_20200717_1137.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.14 on 2020-07-17 11:37
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('versions', '0008_auto_20200625_1114'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='version',
+            name='recommendation_approved',
+            field=models.BooleanField(null=True),
+        ),
```

Comment:
just realised this is missing a RunPython to add a `PromotedApproval` if `version.recommendation_approved`

Reply:
This is still missing ?

Reply:
https://github.com/mozilla/addons-server/pull/14988/commits/d5856a3046e1a08679cecff4b80e7c4eed21278d